### PR TITLE
Add dependabot

### PR DESCRIPTION
**Pull request by nielm**
Dependabot will propose PRs to bring dependancies up to date according to the rules in package.json

(we use [Caret ranges](https://docs.npmjs.com/cli/v6/using-npm/semver#caret-ranges-123-025-004) so this will only propose minor and patch level version updates, not major version updates)

https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuring-dependabot-version-updates

### CODE REVIEW

File: .github/dependabot.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      interval: "weekly"
+  - directory: "/src/forwarder"
+    package-ecosystem: "npm"
+    schedule:
+      interval: "weekly"
+  - directory: "/src/poller"
+    package-ecosystem: "npm"
+    schedule:
+      interval: "weekly"
+  - directory: "/src/poller-core"
```
Comment (henrybell) [inline .github/dependabot.yaml:23]: Should this be /src/poller/poller-core?

File: .github/dependabot.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    package-ecosystem: "npm"
+    schedule:
+      interval: "weekly"
+  - directory: "/src/scaler"
+    package-ecosystem: "npm"
+    schedule:
+      interval: "weekly"
+  - directory: "/src/scaler-core"
```
Comment (henrybell) [inline .github/dependabot.yaml:31]: Should this be /src/scaler/scaler-core?

File: .github/dependabot.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+      interval: "weekly"
+  - directory: "/scaler"
+    package-ecosystem: "docker"
+    schedule:
+      interval: "weekly"
+  - directory: "/forwarder"
+    package-ecosystem: "docker"
+    schedule:
+      interval: "weekly"
+version: 2
```
Comment (henrybell) [inline .github/dependabot.yaml:48]: Do we also want /src in this list for docker?

**Review by henrybell — CHANGES_REQUESTED** [review]

File: .github/dependabot.yaml (hunk 1 above)
Comment (nielm) [reply .github/dependabot.yaml]: fixed

File: .github/dependabot.yaml (hunk 2 above)
Comment (nielm) [reply .github/dependabot.yaml]: fixed

File: .github/dependabot.yaml (hunk 3 above)
Comment (nielm) [reply .github/dependabot.yaml:48]: Fixed. 

Also removed the additional package.json and package-lock.json files

**Review by henrybell — APPROVED** [review]

**Pull request merged**